test/e2e/create_test.go

```diff
@@ -108,10 +173,10 @@ func (tc *testContext) deleteMachineSet(ms *mapi.MachineSet) error {
 
 // waitForWindowsMachines waits for a certain amount of Windows Machines to reach a certain phase
 // if machineCount = 0, it implies we are only waiting for Machines to be deleted and the phase is
-// ignored in this case.
-func (tc *testContext) waitForWindowsMachines(machineCount int, phase string) error {
+// ignored in this case. Returns the set of Machines that matched the provided windowsLabel criteria.
+func (tc *testContext) waitForWindowsMachines(machineCount int, phase string, windowsLabel bool) (*mapi.MachineList, error) {
```

Comment:
What about https://github.com/openshift/windows-machine-config-operator/pull/446#discussion_r633786904?

Reply:
Sorry, missed that.

My concern is that if waitForWindowsMachines takes in a list of machines, createWindowsMachineSet and waitForWindowsMachines are both waiting for a set of Windows machines. waitForWindowsMachines doing what  it does now, and createWindowsMachineSet waiting until Machines matching the label selector are created.

Reply:
Would changing the name of `waitForWindowsMachines` fix the issue you have?

Reply:
Lets discuss this during the meeting.